### PR TITLE
upgrade lhapdf to 6.2.1

### PR DESCRIPTION
**Pull request by mkirsano**

### CODE REVIEW

**Comment by mkirsano** [comment]
Backport of #3415

**Comment by perrozzi** [comment]
@cmsbuild please test

**Comment by smuzaffar** [comment]
+externals

**Comment by perrozzi** [comment]
Hi @davidlange6 can you merge this? 
71x is needed for the MG5_aMC@NLO gridpacks and we need the latest and greatest PDF included here.
So also the timescale for a new CMSSW_7_1_x is becoming very urgent for GEN

File: lhapdf_pdfsetsindex.file [hunk 1]
```diff
@@ -355,7 +355,9 @@
 81150 JR14NNLO20FF 1
 81200 JR14NLO08VF 1
 81250 JR14NLO08FF 1
-82000 LUXqed_plus_PDF4LHC15_nnlo_100 10001
+82000 LUXqed_plus_PDF4LHC15_nnlo_100 10002
+82200 LUXqed17_plus_PDF4LHC15_nnlo_100 10001
+82350 LUXqed17_plus_PDF4LHC15_nnlo_30 20001
```
Comment (perrozzi) [inline lhapdf_pdfsetsindex.file]: @mkirsano can you please crosscheck how many replicas are contained in these PDF sets?
10K or 20K seem to be a little bit too much... Likely is a bug

Comment (perrozzi) [reply lhapdf_pdfsetsindex.file]: looking at the NumMembers for each set in /cvmfs/cms.cern.ch/lhapdf/pdfsets/6.2
these lines need to be changed to
```
82000 LUXqed_plus_PDF4LHC15_nnlo_100 108
82200 LUXqed17_plus_PDF4LHC15_nnlo_100 108
82350 LUXqed17_plus_PDF4LHC15_nnlo_30 38
```

**Comment by perrozzi** [comment]
(previous comments were false alarms, the PR is ready to go)

**Comment by davidlange6** [comment]
+1

**Pull request merged**